skills/dynamo-troubleshoot/scripts/collect_dynamo_debug_bundle.py

```
@@ -0,0 +1,223 @@
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--namespace", "-n", required=True)
+    parser.add_argument(
+        "--deployment-name", help="DynamoGraphDeployment name, if known"
+    )
+    parser.add_argument(
+        "--selector", help="Optional pod selector, for example app=my-app"
+    )
+    parser.add_argument("--outdir", default=f"/tmp/dynamo-debug-{int(time.time())}")
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/dynamo-debug-"

(S108)